### PR TITLE
removed unsed variables

### LINKED ISSUE #2643: unused variables in some activities.
**Describe the bug**

there are some unused variables in some activities.

**Expected behavior**

unused variables should be removed.

**Would you like to work on the issue?**

- [X] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by kk63852**
Fixed #2643 

Changes:
the variables which had no usage in any place were removed.
some of the string constant variable were commented out for future use.

### CODE REVIEW

**Comment by yashk2000** [comment]
@kk63852 please do it more carefully. A lot of the unused variables might be used during app tests. That's why I told you to be very careful. I think it will be better to leave them as they are. We cannot predict all the variables which might be used during tests on our own.

**Comment by kk63852** [comment]
I'll just try fixing the travis build error.

**Comment by yashk2000** [comment]
Ok, try it. But I think it will be better to leave it.

File: app/src/main/java/org/fossasia/phimpme/utilities/Constants.java [hunk 1]
```diff
@@ -34,22 +34,22 @@
 
     public final static String TUMBLR_CONSUMER_KEY = "gOrriv5kT1wOSnZg0u66vXwmo9phPgeXqBlFKOgVQshS4Xe9qh";
     public final static String TUMBLR_CONSUMER_SECRET = "kQFgrWbIsJK3WEabkUPBOR9zZwSZuM3ikOgCpAk4bYs8HqsBTQ";
-    public final static String CALL_BACK_TUMBLR = "http://phimp.me";
+//    public final static String CALL_BACK_TUMBLR = "http://phimp.me";
```
Comment (abishekvashok) [inline app/src/main/java/org/fossasia/phimpme/utilities/Constants.java]: Remove them

Comment (kk63852) [reply app/src/main/java/org/fossasia/phimpme/utilities/Constants.java]: removed. @abishekvashok

**Comment by abishekvashok** [comment]
Thanks

**Pull request merged**